JavaCryptographicArchitecture/src/Cipher.crysl

```diff
@@ -27,6 +29,9 @@ OBJECTS
 	java.nio.ByteBuffer plainBuffer;
 	java.nio.ByteBuffer cipherBuffer;
 	
+	byte[] aad;
```

Comment:
I would suggest a better naming for this, like `aad_byte`.

Reply:
I changed it to your suggestion.